### PR TITLE
fix: add pool id to xyk pool created/destroyed events

### PR DESCRIPTION
**Pull request by enthusiastmartin**

### CODE REVIEW

**Comment by jak-pan** [comment]
@enthusiastmartin can we also add pair_account to the events?

**Comment by mckrava** [comment]
But this commit `fix: add pool id to xyk pool created/destroyed events` adds `shared_token_id` but not `pool id` :)

File: pallets/xyk/src/lib.rs [hunk 1]
```diff
@@ -169,11 +169,11 @@ pub mod pallet {
...
 
-		/// Pool was created. [who, asset a, asset b, initial shares amount]
-		PoolCreated(T::AccountId, AssetId, AssetId, Balance),
+		/// Pool was created. [who, asset a, asset b, initial shares amount, pool id]
+		PoolCreated(T::AccountId, AssetId, AssetId, Balance, AssetId),
 
-		/// Pool was destroyed. [who, asset a, asset b]
-		PoolDestroyed(T::AccountId, AssetId, AssetId),
+		/// Pool was destroyed. [who, asset a, asset b, pool id]
+		PoolDestroyed(T::AccountId, AssetId, AssetId, AssetId),
```
Comment (jak-pan) [inline pallets/xyk/src/lib.rs:176]: this is share token id not exactly pool id -> we should add pool address as well

Comment (enthusiastmartin) [reply pallets/xyk/src/lib.rs]: added

**Review by jak-pan — APPROVED** [review]

**Pull request merged**